nflow-rest-api-common/src/main/java/io/nflow/rest/v1/converter/MaintenanceConverter.java

```diff
@@ -27,7 +31,11 @@ public MaintenanceConfiguration convert(MaintenanceRequest request) {
   }
 
   private ConfigurationItem createConfig(MaintenanceRequestItem config) {
-    return new ConfigurationItem.Builder().setOlderThanPeriod(config.olderThanPeriod).setBatchSize(config.batchSize).build();
+    return new ConfigurationItem.Builder() //
+        .setOlderThanPeriod(config.olderThanPeriod) //
+        .setBatchSize(config.batchSize) //
+        .setWorkflowTypes(ofNullable(config.workflowTypes).orElseGet(Collections::emptySet)) //
```

Comment:
`.orElse(emptySet())` is faster. No need for lambda.

Reply:
I though config.workTypes would usually be present so calling emptySet() could be avoided when it is not really needed.

Reply:
Yes. That is the usual logic. If you would do a `new ArrayList()` then it would be better to use `orElseGet`. But since `emptySet()` is a constant variable there is no harm in passing the constant value. For example `.orElse(-1)` vs `.orElseGet(() -> -5)`